tracing-example/pom.xml

```diff
@@ -37,17 +37,27 @@
         <version.org.jboss.resteasy>6.2.3.Final</version.org.jboss.resteasy>
         <version.org.jboss.resteasy.extensions>2.0.0.Final</version.org.jboss.resteasy.extensions>
         <version.org.junit>5.9.2</version.org.junit>
-        <version.org.wildfly>27.0.1.Final</version.org.wildfly>
         <version.org.wildfly.arquillian>5.0.0.Alpha6</version.org.wildfly.arquillian>
-
+<!--        There is bug in the plugin that prevent from `mvn wildfly:run` currently, please use the command `mvn wildfly:dev` instead. -->
```

Comment:
I think we can leave this off as we'll probably forget to remove it :)

Reply:
yes sir!